src/components/cax-responsibilities.tsx

```diff
@@ -109,9 +123,10 @@ export const ResponsibilitiesCax = ({
   const validatePersonalNote = (note: string) =>
     /^[a-zA-Z][a-zA-Z0-9 !#'$@&%()*+\r\n,\-_./:;=<>?[\]\\^]{0,255}$/.test(note)
 
-  
-    const handleSendInvite = () => {
+
+  const handleSendInvite = () => {
     if (email && validateEmail(email)) {
+      setLoading(true)
```

Comment:
better to make it false after the job. this might keep showing loading in rare conditions.